README.md

```diff
@@ -843,7 +844,6 @@ Table of Contents
    * [kanbanflow.com](https://kanbanflow.com/) — Board-based project management. Free, premium version with more options
    * [kanbantool.com](https://kanbantool.com/) — Kanban board-based project management. Free, paid plans with more options
    * [kanrails.com](https://kanrails.com/) — Kanban board-based project management. Free for 3 collaborators, 2 projects and 5 tracks. Paid plans available for unlimited collaborators, projects and tracks.
-   * [Kitemaker.co](https://kitemaker.co) - Collaborate through all phases of the product development process and keep track of work across Slack, Discord, Figma, and Github. Unlimited users, unlimited spaces. Free plan up to 250 work items.
```

Comment:
this one should stay (just merged, perhaps some weird conflict here)

Reply:
I was wondering about this, closed/reopened PR

Reply:
Thought I accidentally deleted that line

Reply:
lmk we can recreate new PR

Reply:
You can also just send a 2nd commit to the same PR to put it back, thats fine too - which is easier for you, new PR or update this one

Reply:
on it

Reply:
done - https://github.com/ripienaar/free-for-dev/pull/1812/commits/f41a5a310fcde6fe4ec002abe1367b05cb5a4abc

Reply:
added - https://github.com/ripienaar/free-for-dev/commit/f41a5a310fcde6fe4ec002abe1367b05cb5a4abc

![image](https://user-images.githubusercontent.com/45214/108901845-3c490d80-75d0-11eb-861c-6f5194dc5e61.png)
![image](https://user-images.githubusercontent.com/45214/108901865-410dc180-75d0-11eb-93f1-2d5708fd13c0.png)